src/components/UserInfo/UserInfo.jsx

```diff
@@ -2,0 +1,7 @@
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user.email}`}>
```

Comment:
The `href` attribute for the email link is constructed correctly using a template literal. Ensure that `user.email` is a valid email address to avoid issues with the mailto link.